website/docs/gen-ai/inference/Llama2.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Llama-2 on EKS
+sidebar_position: 1
+---
+import CollapsibleContent from '../../../src/components/CollapsibleContent';
+
+
+:::danger
+
+Note: Use of this Llama-2 model is governed by the Meta license.
+In order to download the model weights and tokenizer, please visit the [website](https://ai.meta.com/) and accept our License before requesting access here.
```

Comment:
"accept our License before requesting access here." -> accept the license before requesting access.